tests/int/test_headers.py

```diff
@@ -0,0 +1,174 @@
+import pytest
+from pytest_check import check
+import requests
+import os
+import asyncio
+from api_test_utils.oauth_helper import OauthHelper
+from api_test_utils.apigee_api_apps import ApigeeApiDeveloperApps
+from api_test_utils.apigee_api_products import ApigeeApiProducts
+from time import time
+
+
+ENVIRONMENT = os.environ['ENVIRONMENT']
+SERVICE_BASE_PATH = os.environ['SERVICE_BASE_PATH']
+SERVICE_NAME = os.environ['FULLY_QUALIFIED_SERVICE_NAME']
+BASE_URL = f'https://{ENVIRONMENT}.api.service.nhs.uk/{SERVICE_BASE_PATH}'
```

Comment:
This is also happening in a config file in another folder. If not now then as you develop more tests consider moving some of this and stuff like fixtures out into config and conftest files. See ImmHist - https://github.com/NHSDigital/immunisation-history-api/tree/master/tests

Reply:
I have now cleaned up the whole config and implemented everything as fixtures in conftest.py